dags/hivemind_etl_helpers/src/db/telegram/utils/module.py

```
@@ -0,0 +1,119 @@
+        --------
+        existence : bool
+            True, if a community module is already set
+            False, if there's no module related to the community
+        """
+        document = self._client[self.database][self.collection].find_one(
+            {"community": ObjectId(self.community_id)},
+            {
+                "_id": 1,
+            },
+        )
+        return True if document else False
```

Ruff

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)